src/.editorconfig

```diff
@@ -166,7 +166,7 @@ dotnet_diagnostic.CA2002.severity = warning
 dotnet_diagnostic.CA2003.severity = none
 dotnet_diagnostic.CA2004.severity = none
 dotnet_diagnostic.CA2006.severity = none
-dotnet_diagnostic.CA2007.severity = error
+dotnet_diagnostic.CA2007.severity = none
```

Comment:
This needs to stay at `warning` at least, for the reasons I mentioned in the previous PR.